Library/Homebrew/keg.rb

```diff
@@ -339,14 +339,14 @@ def to_formula
   end
 
   def installed_dependents
+    return [] unless linked?
```

Comment:
So, a non-opt-linked `keg_only` keg isn't considered to have any installed dependents?